sources/packages/backend/apps/queue-consumers/src/processors/assessment/_tests_/cancel-application-assessment.processor.e2e-spec.ts

```diff
@@ -73,6 +79,8 @@ describe(
       // Adjust assessment.
       const studentAssessment = application.currentAssessment;
       studentAssessment.assessmentWorkflowId = workflowInstanceId;
+      studentAssessment.studentAssessmentStatus =
```

Comment:
Minor, the factory can have a property like `assessmentInitalValues` to set the assessment status